doc(Partner SDK): Add the documentation related to V2 proto changes (#82)
fivetran/fivetran_sdk: tools/destination-tester/README.md

    ```
        docker pull us-docker.pkg.dev/build-286712/public-docker-us/sdktesters/sdk-tester   
    ```

> > Note: If using V2 proto versions, please use latest docker image from [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) Google Artifact Registry.

Reviewer: ```suggestion
> NOTE: If using V2 proto versions, use latest docker image of the [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) artifact in Google Artifact Registry.
```
?

Author: Yes, this looks good.

> > NOTE: If using V2 proto versions, use latest docker image of the [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) artifact in Google Artifact Registry.

Reviewer 2: Should this be be updated to
`If using V2 proto versions`
or
`For V2 proto versions` ?

Reviewer: I am OK with either, both are correct and clear.
@fivetran-satvikpatil feel free to accept Manul's suggestion or leave as is.
```suggestion
> NOTE: For V2 proto versions, use latest docker image of the [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) artifact in Google Artifact Registry.
```

> > NOTE: If using V2 proto versions, use latest docker image of the [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) artifact in Google Artifact Registry.

Reviewer: ```suggestion
> NOTE: If using V2 proto versions, use the latest docker image of the [public-docker-us/sdktesters-v2/sdk-tester](https://console.cloud.google.com/artifacts/docker/build-286712/us/public-docker-us/sdktesters-v2%2Fsdk-tester?invt=Abm4dQ&inv=1) artifact in Google Artifact Registry.
```

The tester by default creates a schema named `tester`. This option allows the tester to run with a custom schema name by specifying `--schema-name <custom_schema_name>` where `<custom_schema_name>` is your custom schema name.

#### --disable-operation-delay
The tester by default adds a delay to operation for real-time simulation. Specifying this argument disables the delay.

#### --batch-file-type

> We now support both CSV and PARQUET batch files. If this argument is not provided, testers will generate CSV batch files.

Reviewer 2: @5tran-alexil Is better to mention that default value is CSV.

We now support batch files in both CSV and PARQUET formats, with CSV set as the default option.

Reviewer: @manjutapali I don't have a strong opinion here yet I believe the "default option" is less clear than "If this parameter is not provided" - the current wording describes on what conditions the default CSV format is used.
